docs/samples/s3/tensorflow_s3.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "mnist-s3"
+spec:
+  default:
+    tensorflow:
+      # serviceAccountName: sa   // overwrite default service account
```

Comment:
Can you delete this?

Reply:
Rather, can we wire it up such that the service account in the docs points directly to this?

Reply:
Done